README.md

```diff
@@ -45,6 +52,21 @@ message causing performance issues.
 
 To view logs across all log streams it is recommended to use the [Logs Insight](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/AnalyzingLogData.html) feature for Cloud Watch Logs.
 
+### Setting new Log Group Retention Policy
+
+These libraries support setting a [log retention policy](https://docs.aws.amazon.com/managedservices/latest/userguide/log-customize-retention.html) 
+on any CloudWatch Log Groups which they create.  This feature is enabled using the NewLogGroupRetentionInDays configuration property. 
+The DisableLogGroupCreation configuration property must not be set to true. Retention policies configured in this manner 
+are only applied to _new_ Log Groups created directly by these libraries. By default no retention policy is applied to newly created Log Groups.
+
+
+Note that any value of NewLogGroupRetentionInDays which is not one of the below is a configuration error which will result
+in a non-fatal error applying the policy. The application and logging will continue however no retention policy will be applied.
+ 
+```csharp
+null, 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 2192, 2557, 2922, 3288, 3653
```

Comment:
`1096` value is missing from the possible values here and at `NewLogGroupRetentionInDays` comments
https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax:~:text=Required%3A%20Yes-,retentionInDays,-The%20number%20of

Can you include it in the possible values list or better add a link to to [retentionInDays](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html#API_PutRetentionPolicy_RequestSyntax) from the docs

Reply:
@Will-Boulton Please check the above review comment and make the necessary changes so that we could approve this PR.

Thanks,
Ashish

Reply:
I have made the requested change to add a link to the relevant section of the CloudWatch documentation. Apologies for the delay. 

Thanks both